aws-ecs/20-rds.tf

```diff
@@ -0,0 +1,88 @@
+resource "aws_security_group" "rds" {
+  name   = "rds-sg"
+  vpc_id = local.vpc_id
+
+  ingress {
+    description = "Allow postgress ingress from private subnets"
+    from_port   = 5432
+    to_port     = 5432
+    protocol    = "tcp"
+    cidr_blocks = local.allowed_cidr_blocks
+  }
+
+  tags = {
+    "Name" = "rds_sg"
+  }
+}
+
+resource "aws_db_subnet_group" "rds" {
+  name       = "pvault"
+  subnet_ids = local.database_subnet_ids
+
+  tags = {
+    Name = "pvault_subnet_group"
```

Comment:
I think we can remove the various tags, in favor a [default_tags](https://www.hashicorp.com/blog/default-tags-in-the-terraform-aws-provider) with some basic tags like:
* `Terraform: true`
* `Service: Piiano Vault`

@jackishere wdyt?